manifests/repo.pp

```diff
@@ -17,17 +17,22 @@
         }
       }
       'Debian': {
-        $release_url = $facts['os']['name'] ? {
-          'Debian' => "http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/Debian_${major_release}.0",
-          'Ubuntu' => "http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_${major_release}",
+        $release_url_suffix = $facts['os']['name'] ? {
+          'debian' => $major_release ? {
```

Comment:
any particular reason why you changed this to lowercase?

Reply:
none, indeed.

Do you want I capitalize `debian` and `ubuntu` ?

Reply:
yes please, IIRC something broke once upon a time with lowercase osnames

Reply:
Fixed, thanks for review.